### DISCUSSION TITLE
Tailwind module removes default markdown styling?

### DISCUSSION THREAD

**Discussion by AdamBD** [nuxt/content#2739] [Q&A]
Hello,

I have added nuxt content to my site to create a blog. Looking at the demo of nuxt content on the main website, I see that the markdown respects the default styling of the elements the markdown is translated into.

However when I render the pages the elements have no styling, is this because of the tailwind module that removes all defaults?

What is the best way to style these pages then? Usually I am doing it via scoped styling in the page components. Do you need to add in global styles for the markdown pages to pick them up?

**Comment by HuakunShen** [comment]
@AdamBD 
Do you have an answer now? I am also using tailwind with content, but all styles from Markdown are removed by tailwind.

**Comment by HuakunShen** [comment]
I think we have to use Tailwind Typography plugin and the `prose` class.

This Nuxt UI Pro SaaS template uses Tailwind and content (https://saas-template.nuxt.dev/blog/nuxt-ui)

It's blog isn't affected by Tailwind.
I found `<UPageBody prose>` in the code. Without this, tailwind will remove the default styling.
https://github.com/nuxt-ui-pro/saas/blob/5ffdd6b38bad790e553be31d1ecb9fff68fb3b97/app/pages/blog/%5Bslug%5D.vue#L82

`UPageBody` belongs to a private library Nuxt UI Pro, but based on the docs https://ui.nuxt.com/pro/components/page-body#config, I believe the typography plugin is used.

And I  tested, it works when I swap `UPageBody` with `prose`
